### PR TITLE
feature 33785 - Added a new section to the account page

### PR DESCRIPTION
**Pull request by Rick-Ritense**
Added a new GraphQL query for retrieving the number of people living on an address

Added configuration to show/hide the new section

Added a button that allows a person to request an address research. The URL for this button is configurable

### CODE REVIEW

File: packages/app/src/components/app/app.tsx [hunk 1]
```diff
@@ -89,7 +89,12 @@ const pages: Array<PortalPage> = [
   },
   {
     icon: <UserIcon />,
-    pageComponent: <AccountPage />,
+    pageComponent: (
+      <AccountPage
+        showInhabitantAmount={config.SHOW_INHABITANT_AMOUNT === 'true'}
```
Comment (mritense) [inline packages/app/src/components/app/app.tsx:94]: Don't forget to move this logic to the component

File: packages/user-interface/src/pages/account/account-page.tsx [hunk 2]
```diff
@@ -13,7 +17,14 @@ import {
...
 } from '../../utils/person-data';
 
-const AccountPage = () => {
+interface AccountPageProps {
+  showInhabitantAmount?: boolean;
+  addressResearchUrl?: string;
+}
+
+const AccountPage: FC<AccountPageProps> = ({showInhabitantAmount, addressResearchUrl}) => {
+  console.log(`showInhabitantAmount: ${showInhabitantAmount}`);
```
Comment (mritense) [inline packages/user-interface/src/pages/account/account-page.tsx:26]: Please remove console logs

File: packages/user-interface/src/pages/account/account-page.tsx [hunk 3]
```diff
@@ -26,9 +37,21 @@ const AccountPage = () => {
     refetch: personRefetch,
   } = useGetPersoonDataQuery();
 
+  const {
+    data: bewonerAantalData,
+    loading: bewonersAantalLoading,
+    refetch: bewonersAantalRefetch,
+  } = useGetBewonersAantalQuery();
+
+  function openAddressInvestigation() {
```
Comment (mritense) [inline packages/user-interface/src/pages/account/account-page.tsx:46]: Please rewrite as arrow function with void return type

File: packages/user-interface/src/pages/account/account-page.tsx (hunk 2 above)
Comment (Rick-Ritense) [reply packages/user-interface/src/pages/account/account-page.tsx:26]: Will do

**Review by mritense — APPROVED** [review]

**Pull request merged**